plugins/dynamic-plugins-info/CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Setting up the development environment for Dynamic Plugins Info plugin
+
+In [Backstage plugin terminology](https://backstage.io/docs/local-dev/cli-build-system#package-roles), the Dynamic Plugins Info plugin is a front-end plugin. You can start a live development session from the repository root using the following command:
+
+```console
+yarn workspace @janus-idp/backstage-plugin-dynamic-plugins-info run start
```

Comment:
@janus-idp/backstage-plugin-dynamic-plugins-info might not be needed right or if this file needs an update too?

Reply:
Yep, I think I should probably clean up all of this for this PR.